kolibri/plugins/coach/assets/src/views/recent-page/index.vue

```diff
@@ -1,62 +1,33 @@
 <template>
 
-  <div id="recent">
-    <channel-list v-if="channelNotSelected" />
+  <div>
+    <h1>{{ $tr('title') }}</h1>
+    <div v-if="pageState.channelId">
+      <report-list :reports="pageState.reports" />
+    </div>
     <div v-else>
-      <table>
-        <thead>
-          <tr>
-            <th> Name </th>
-            <th> Progress </th>
-          </tr>
-        </thead>
-
-        <tbody>
-          <tr v-for="report in reports">
-            <td>
-              {{report.title}}
-            </td>
-            <td>
-              {{report.progress[0].total_progress}}
-            </td>
-          </tr>
-        </tbody>
-
-      </table>
+      <channel-list />
     </div>
+
   </div>
 
 </template>
 
 
 <script>
 
-  const PageNames = require('../../constants').PageNames;
-
   module.exports = {
     $trNameSpace: 'coachRecentPage',
     $trs: {
-      recentPage: 'Recent Page',
-      header: 'Recent Activity - ',
-      pageDescription: 'Content your Learners have recently completed or mastered',
-      name: 'Name',
-      progress: 'Progress',
-      noRecentProgressExist: 'No recent progress.'
+      title: 'Recent Activity',
     },
     components: {
       'channel-list': require('./channel-list'),
-    },
-    computed: {
-      channelNotSelected() {
-        return this.subPageName === PageNames.RECENT_CHANNEL_SELECT;
```

Comment:
I was trying to keep `PageNames` as consistent as possible, and it was weird to me to have both `RECENT_CHANNEL_SELECT` and `RECENT`.

I can see potentially making constants for subpage names, but then they should be in a different namespace from `PageNames`